seqr/views/utils/dataset_utils.py

```diff
@@ -427,12 +432,16 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
     }
 
     prefetch_related_objects(list(samples_to_load.keys()), 'individual__family__project')
-    projects = {sample.individual.family.project.name for sample in samples_to_load}
-    project_names = ', '.join(sorted(projects))
-    message = f'Attempted data loading for {len(samples_to_load)} RNA-seq samples in the following {len(projects)} projects: {project_names}'
+    project_samples = defaultdict(list)
+    for sample in samples_to_load:
+        project_samples[sample.individual.family.project].append(sample)
```

Comment:
Will the performance be better to use values_list() to get the `project.guid` and `project.name` for the samples directly?